### ISSUE TITLE
State of this project?

### ISSUE THREAD

**Issue by hbruch** [afimb/chouette#60]
Bonjour,

merci beaucoup for providing chouette. As the last activity in this project was more than a year ago and various dependencies are outdated: Is this project still actively maintained? If not, is there a successor project?

Cordialement,
Holger

**Comment by Aurige** [comment]
Yes this repository is now dormant and the new version is https://enroute.mobi/produits/chouette/ (also https://github.com/enroute-mobi )
